docs/sources/getting-started/getting-started-elasticsearch.md

```diff
@@ -0,0 +1,123 @@
+Check also out [Using Elasticsearch in Grafana](/docs/grafana/latest/datasources/elasticsearch/) for a more detailed list of all available configuration options for Elasticsearch.
+
+To add the Elasticsearch data source:
+
+1. In the Grafana side menu, hover your cursor over the **Configuration** (gear) icon and then click **Data Sources**.
+1. Select the **Elasticsearch** option.
+1. Click **Add data source** in the top right header to open the configuration page.
+1. Enter the information specified in the tables below, then click **Save & Test**.
+
+> **Note**: We'll assume you used the instructions from Grafana's devenv for simplicity, if this is not the case you may have to adapt the values in the tables below to match your use-case.
+
+### General Settings
+
+| Name   | Description                                                                           |
+| ------ | ------------------------------------------------------------------------------------- |
+| `Name` | The data source name. This is how you refer to the data source in panels and queries. |
+
+### HTTP Settings
+
+| Name     | Description                                                                                               | Value                    |
+| -------- | --------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `URL`    | The IP address/hostname and optional port of your Elasticsearch instance.                                 | `http://localhost:12200` |
+| `Access` | whether to connect to your Elasticsearch instance from the browser or through Grafana's backend (Server). | `Server (default)`       |
+
+### Elasticsearch details
+
+| Name              | Description                                                                | Value                  |
+| ----------------- | -------------------------------------------------------------------------- | ---------------------- |
+| `Index name`      | The index pattern used to identify indices you want to query with grafana. | `[metrics-]YYYY.MM.DD` |
+| `Pattern`         | The time pattern for which the above index will be be interpolated.        | `Daily`                |
+| `Time field name` | The name of the field that identiefies the timestamp of your documents.    | `@timestamp`           |
+| `Version`         | The Elasticsearch version you are running.                                 | `7.0+`                 |
+
+Once you click **Save & Test** the following message will appear, confirming that Grafana is able to communicate with your Elasticsearch instance:
+
+![Elasticsearch confirmation message](/img/docs/getting-started/elasticsearch/confirmation-7-4.png)
+
+## Step 4. Create your first Elasticsearch backed dashboard
+
+1. In the Grafana side menu, hover your cursor over the **Create** (plus) icon and then click **Dashboard**.
+1. Click the **Add new panel** button.
```

Comment:
They appear indented. Can you double-check?

Reply:
Appears like this when building locally

![Screenshot 2021-02-04 at 16 10 36](https://user-images.githubusercontent.com/1170767/106921147-a07c5e00-6703-11eb-9c53-187893c58da6.png)

Reply:
They look good to me.

Reply:
I suggest adding a sentence after the heading to help give the user an idea of what they'll be doing. Keep in mind that this topic is aimed at a Grafana beginner.